schema.graphql

```diff
@@ -366,11 +365,12 @@ type FilledOrdersEvent @entity {
   fee: BigInt!
 }
 
-# LOP
-
+# LOP -> Event
 type OrderFilledEvent @entity {
-  "Transaction Hash"
+  "Transaction hash + Order hash"
```

Comment:
Here is the fix, the ID is now a combination of `txHash` and `orderHash`, this works for all the scenarios of multiple events o an order in more than one event.